### DISCUSSION TITLE
Weight on teammates.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#96] [Ideas]
Increasing or decreasing weight on teammates? idk if this would work, sounds like a host only thing even if its possible.

**Comment by winstxnhdw** [comment]
Don't think it's possible. If all scraps have a pre-defined weight, this isn't possible, but if they are randomised, we might be able to pick it up, set it's weight and ask a teammate to pick it up. That's the best we can do.

**Reply by yujispin990** [reply]
oh wait actually that's really nice! hopefully you would take it into consideration.

**Reply by winstxnhdw** [reply]
Will have to first check if this is true.

>  If all scraps have a pre-defined weight, this isn't possible

**Reply by yujispin990** [reply]
Please and thank you sir!
